Fix index name in bitmap filtering docs

russcam opened the pull request:
### Description

Fix the name of the index used in some examples so that it is consistent with the surrounding documentation.

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -277,7 +277,7 @@ PUT /products
 
 ```json
-PUT students/_doc/1
+PUT products/_doc/1
````
```suggestion
PUT /products/_doc/1
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -286,7 +286,7 @@ PUT students/_doc/1
 
 ```json
-PUT students/_doc/2
+PUT products/_doc/2
````
```suggestion
PUT /products/_doc/2
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -295,7 +295,7 @@ PUT students/_doc/2
 
 ```json
-PUT students/_doc/3
+PUT products/_doc/3
````
```suggestion
PUT /products/_doc/3
```

kolchfa-aws reviewed (approved):
Thank you for making the docs better, @russcam! A small suggestion to keep the requests uniform.

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -83,6 +86,7 @@ PUT students/_doc/2
   "student_id" : "222"
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -92,6 +96,7 @@ PUT students/_doc/3
   "student_id" : "333"
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -103,6 +108,7 @@ PUT classes/_doc/101
   "enrolled" : ["111" , "222"]
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -121,6 +127,7 @@ GET students/_search
   }
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -178,6 +185,7 @@ PUT classes/_doc/102
   }
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -196,6 +204,7 @@ GET students/_search
   }
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
```diff
@@ -253,12 +262,13 @@ Parameter | Data type | Description
 `boost` | Floating-point | A floating-point value that specifies the weight of this field toward the relevance score. Values above 1.0 increase the field’s relevance. Values between 0.0 and 1.0 decrease the field’s relevance. Default is 1.0.
 
 ## Bitmap filtering
+
```
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -272,35 +282,39 @@ PUT /products
   }
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -272,35 +282,39 @@ PUT /products
   "product_id" : "111"
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -272,35 +282,39 @@ PUT /products
   "product_id" : "333"
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -318,6 +332,7 @@ PUT /customers
   }
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -338,6 +353,7 @@ encoded_bm_str = encoded.decode('utf-8')
 # Print the encoded bitmap
 print(f"Encoded Bitmap: {encoded_bm_str}")
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -383,4 +401,5 @@ POST /products/_search
   }
 }
 ```
+
````
```suggestion
```

kolchfa-aws reviewed (approved):
LGTM

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -22,6 +22,7 @@ GET shakespeare/_search
   }
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -63,6 +64,7 @@ PUT students
   }
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented on _query-dsl/term/terms.md:
````diff
@@ -74,6 +76,7 @@ PUT students/_doc/1
   "student_id" : "111"
 }
 ```
+
````
```suggestion
```

kolchfa-aws commented:
@russcam Could you fix DCO so we can merge this PR? I just removed the extra lines between the code blocks and the includes. Thanks!

russcam commented:
@kolchfa-aws done

kolchfa-aws commented:
Hey @russcam, looks like DCO is still failing, and you've picked up some extraneous commits. Could you please rebase and sign the commits that we want that are not signed? https://github.com/opensearch-project/documentation-website/pull/9032/checks?check_run_id=35404113474 Thanks so much!

russcam commented:
I can fix the issue that DCO has, but it requires me to sign commits that I haven't made (2604e82, ff96083). These extraneous commits are ones that have come in from the rebasing and merging that's been applied to the branch.

I was in the process of addressing the review feedback, when GitHub was having issues, and these changes were made. A simple PR has been overcomplicated 🙂

kolchfa-aws commented:
I think this may require an interactive rebase. Otherwise, do you want to close this PR and just raise a new one with the changes: might be easier.

kolchfa-aws commented:
@russcam Do you want to close this PR and open a new one with the changes? Might be easier 😄

kolchfa-aws commented:
@russcam I fixed the extra commits. Merging the PR now 😄

russcam commented:
Thanks @kolchfa-aws 🙏
